_rules/additional-content-triggered-on-hover-is-hoverable-ep1s13.md

```diff
@@ -0,0 +1,170 @@
+---
+id: ep1s13
+name: Additional content triggered on hover is hoverable
+rule_type: atomic
+description: |
+  This rule checks that any additional content that was triggered by hovering an element remains available to the user while the mouse pointer is over the element or the additional content
+accessibility_requirements:
+  wcag21:1.4.13: # Content on Hover or Focus (AA)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+  - Mouse pointer
+acknowledgments:
+  authors:
+    - Carlos Duarte
+---
+
+## Applicability
+
+The rule applies to any element that when [hovered][] causes [visible changes in content][] and for the [area affected by the visible changes][area affected by visible changes], determined after 5 seconds of [hovering][hovered] the element, all the following are true:
```

Comment:
I think this shows the problem with this rule; this SC doesn't apply to presentational changes, just to changes in content, whereas the definition you are using includes presentational changes.

```suggestion
The rule applies to any element that when [hovered][] causes [visible changes in presentation or content][] and for the [area affected by the visible changes][area affected by visible changes], determined after 5 seconds of [hovering][hovered] the element, all the following are true:
```